src/main/java/com/_up/megastore/services/implementations/SizeService.java

```diff
@@ -47,12 +47,12 @@ public SizeResponse updateSize(UUID sizeId,UpdateSizeRequest updateSizeRequest){
     @Override
     public SizeResponse restoreSize(UUID sizeId){
         Size size = findSizeByIdOrThrowException(sizeId);
-        validateSizeForRestoration(size);
+        ifSizeIsAlreadyDeletedThrowException(size);
         size.setDeleted(false);
         return SizeMapper.toSizeResponse(sizeRepository.save(size));
     }
 
-    public void validateSizeForRestoration(Size size) {
+    public void ifSizeIsAlreadyDeletedThrowException(Size size) {
```

Comment:
El método está mal redactado, estás diciendo que si está eliminado tirás una excepción cuando debería ser lo contrario
```suggestion
    public void ifSizeIsNotDeletedThrowException(Size size) {
```